### DISCUSSION TITLE
What does it mean to 'reload' the app on Expo Go?

### DISCUSSION THREAD

**Discussion by azizsaad** [expo/expo#23473] [Archived: Questions]
On the Expo Go app, you can open the settings as shown below:

<img width="507" alt="Screenshot 2023-07-11 at 20 52 05" src="https://github.com/expo/expo/assets/46464104/e3080712-e063-4dff-8f03-91ce1584dee7">

What does it mean to 'reload' the app on Expo Go, in terms of the state of an iOS mobile?

i.e. does the app go from being Inactive to Active?

**Comment by amandeepmittal** [comment]
The `reload` button in the developer menu reloads your project in Expo Go (or if using development build). In most cases, it is not necessary since Fast refresh is enabled by default. You read more about different options available in developer menu in the docs here: https://docs.expo.dev/debugging/tools/#developer-menu

**Reply by azizsaad** [reply]
Hey!

Thank you for your reply and the link.

My question was moreso about the state of the app i.e., when you Reload the app, does the app from `Active` -> `Inactive/Background` -> `Active`? That is what I was not sure off.

**Reply by azizsaad** [reply]
Just realised you replied to this in the office hours just now 😆 Thank you for your answer! To confirm on here, reloading does cause the app to terminate and restart fresh, i.e. local state refreshes
